i18n/zh-cn/user-docs/kubeblocks-for-starrocks/manage-starrocks.md

```diff
@@ -6,61 +6,152 @@ sidebar_position: 1
 sidebar_label: 用 KubeBlocks 管理 StarRocks
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # 用 KubeBlocks 管理 StarRocks
 
 StarRocks 是一款高性能分析型数据仓库，使用向量化、MPP 架构、CBO、智能物化视图、可实时更新的列式存储引擎等技术实现多维、实时、高并发的数据分析。
 
-KubeBlocks supports the management of StarRocks.
+本文档展示了如何通过 kbcli、kubectl 或 YAML 文件等当时创建和管理 StarRocks 集群。您可以在 [GitHub 仓库](https://github.com/apecloud/kubeblocks-addons/tree/release-0.9/examples/starrocks)查看 YAML 示例。
 
 ## 开始之前
 
-- [安装 kbcli](./../installation/install-with-kbcli/install-kbcli.md)。
-- [安装 KubeBlocks](./../installation/install-with-kbcli/install-kubeblocks-with-kbcli.md)。
-- [安装并启用 starrocks 引擎](./../overview/database-engines-supported.md#使用引擎)。
+- 如果您想通过 `kbcli` 创建并连接 StarRocks 集群，请先[安装 kbcli](./../installation/install-with-kbcli/install-kbcli.md)。
+- 安装 KubeBlocks，可通过 [kbcli](./../installation/install-with-kbcli/install-kubeblocks-with-kbcli.md) 或 [Helm](./../installation/install-with-helm/install-kubeblocks.md) 安装。
+- 安装并启用 starrocks 引擎，可通过 [kbcli](./../installation/install-with-kbcli/install-addons.md) 或 [Helm](./../installation/install-with-helm/install-addons.md) 操作。
 
 ## 创建集群
 
 ***步骤：***
 
-1. 执行以下命令，创建 StarRocks 集群。
-
-   ```bash
-   kbcli cluster create mycluster --cluster-definition=starrocks
-   ```
+<Tabs>
 
-   您可使用 `--set` 指定 CPU、memory、存储的值。
+<TabItem value="kubectl" label="kubectl" default>
 
-   ```bash
-   kbcli cluster create mycluster --cluster-definition=starrocks --set cpu=1,memory=2Gi,storage=10Gi
-   ```
+KubeBlocks 通过 `Cluster` 定义集群。以下是创建 StarRocks 集群的示例。Pod 默认分布在不同节点。但如果您只有一个节点可用于部署集群，可将 `spec.affinity.topologyKeys` 设置为 `null`。
 
 :::note
 
-执行以下命令，查看更多集群创建的选项和默认值。
-  
+生产环境中，不建议将所有副本部署在同一个节点上，因为这可能会降低集群的可用性。
+
+:::
+
+```yaml
+cat <<EOF | kubectl apply -f -
```

Comment:
pls check the StarRocks exampes here. against that of  v0.9 in kb-addons examples.

Reply:
> pls check the StarRocks exampes here. against that of v0.9 in kb-addons examples.

As discussed, StarRocks cluster yaml will be updated later.